src/io/iter_csv.cc

```diff
@@ -187,10 +187,10 @@ Examples::
 
   // Creates a `CSVIter` with `round_batch`=False.
   CSVIter = mx.io.CSVIter(data_csv = 'data/data.csv', data_shape = (3,),
-  batch_size = 3, round_batch=True)
+  batch_size = 3, round_batch=False)
 
-  // Contents of two batches read from the above iterator in both passes after calling
-  // `reset` method before second pass is as follows:
+  // Contents of two batches read from the above iterator in both passes, after calling
+  // `reset` method before second pass, is as follows:
   [[1.  2.  3.]
   [2.  3.  4.]
   [3.  4.  5.]]
```

Comment:
I'm comparing line 172 and 194. So the result of first pass is the same no matter what value `round_batch` is set?